Updated the bitcoin installation instructions

ErikDeSmedt opened the pull request:
It turns out that bitcoin-core stopped maintaining the ppa.
I have updated the instructions to download form BitcoinCore instead.

This is also mentioned in  [issue 143](https://github.com/ElementsProject/elementsproject.github.io/issues/143).

ErikDeSmedt committed:
Updated bitcoin installation instructions

It turns out that bitcoin-core stopped maintaining the ppa.
I have updated the instructions to download form BitcoinCore instead.

wintercooled commented:
Nice one. I will review this now. Thanks.

wintercooled commented:
Make sure you'll download the binaries
Can you change this to -> "Make sure you download the binaries"

SHA256SUM.asc-file.

-> SHA256SUM.asc file.

Instances of SHA256SUM should I think be:
-> SHA256SUMS

/usr/bin/bitcoind
Should this not be /usr/local/bin/bitcoind ?

ErikDeSmedt committed:
Corrected filenames

As commented by mathew
- you should download the SHA256SUMS-file (extra s)
- bitcoind should be in /usr/bin/local/bitcoind

ErikDeSmedt committed:
Grammar of installing-bitcoin.md

Change phrasing of download instructions

ErikDeSmedt committed:
Reference bitcoin PGP-keys in installat instructions

ErikDeSmedt commented:
I have added a short explanation + link to the trusted keys

wtogami commented:
I recommend exporting those keys as ascii armored text and including them in a folder in this repo.
